apps/web/src/components/layout/components/PageContainer.tsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import { Container } from '../../../design-system';
 import PageMeta from './PageMeta';
+import { ContainerProps } from '@mantine/core';
 
-function PageContainer({ children }: { children: React.ReactNode }) {
+export type PageContainerProps = { fullSize?: boolean } & ContainerProps;
+
+function PageContainer({ children, fullSize, ...props }: PageContainerProps) {
   return (
-    <Container pl={0} pr={0} fluid>
+    <Container pl={0} pr={0} fluid {...props} style={fullSize ? { minHeight: '100%', display: 'flex' } : {}}>
```

Comment:
not sure if it is the best way to do so, added it as a temp solution so other pages won't be affected.